tapdb/sqlc/migrations/000002_assets.up.sql

```diff
@@ -107,7 +109,9 @@ CREATE TABLE IF NOT EXISTS asset_groups (
 CREATE TABLE IF NOT EXISTS asset_group_sigs (
     sig_id INTEGER PRIMARY KEY,
 
-    genesis_sig BLOB NOT NULL, 
+    -- The witness stack can contain either a single Schnorr signature for key
+    -- spends of the tweaked group key, or a more complex script witness.
+    witness_stack BLOB NOT NULL,
```

Comment:
I think let's go ahead and rename now? Don't see much of a benefit of doing it later. Particularly since we'll merge this into a side branch that doesn't need to be fully working until we go to merge things back into master.

Reply:
Yup fair enough.

Reply:
We assumed that with the rename you meant also renaming the table from `asset_group_sigs` to `asset_group_witnesses`? That's addressed in the latest push.